CHANGELOG.md

```diff
@@ -5,6 +5,126 @@ All notable changes to this project will be documented in this file. See
 
 ## [4.0.0](https://github.com/taiga-family/taiga-ui/compare/v4.0.0-rc.10...v4.0.0) (2024-08-09)
 
+- Update to Angular 16 (https://github.com/taiga-family/taiga-ui/pull/6966).
+- Update supported browser versions (https://github.com/taiga-family/taiga-ui/pull/6818).
+- Rename **all** entities: drop `Module` / `Component` / `Directive` postfix.
+- Use deep entrypoints in **all** imports paths inside packages for microfrontends tree-shaking support.
+- **@taiga-ui/legacy** create new package ([#7501](https://github.com/taiga-family/taiga-ui/pull/7501)).
+- **@taiga-ui/browserslist-config** create new package (https://github.com/taiga-family/taiga-ui/pull/5344).
+- **@taiga-ui/core:**
+  - `Textfield` add new component ([#6298](https://github.com/taiga-family/taiga-ui/pull/6298)).
+  - `Icon` add new component ([#5872](https://github.com/taiga-family/taiga-ui/pull/5872)). `Icons` add new directive
+    ([#6053](https://github.com/taiga-family/taiga-ui/pull/6053)) `Icon` add new pipe
+    (https://github.com/taiga-family/taiga-ui/pull/7244).
+  - `Icons` new approach **based on mask** ([#7752](https://github.com/taiga-family/taiga-ui/pull/7752),
+    [#7658](https://github.com/taiga-family/taiga-ui/pull/7658),
+    [#7714](https://github.com/taiga-family/taiga-ui/pull/7714)).
+  - `Link` uses `Icon` instead of legacy `Svg` ([#7056](https://github.com/taiga-family/taiga-ui/pull/7056)). `Link`
+    refactor to use `Icons` directive ([#7170](https://github.com/taiga-family/taiga-ui/pull/7170)).
+  - `Flag` add new pipe ([#3725](https://github.com/taiga-family/taiga-ui/pull/3725)).
```

Comment:
It already exists in `v3`:
https://taiga-ui.dev/v3/pipes/flag

Reply:
removed